app/views/addons/_form.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= form_with model: @addon,class:"form-horizontal" do |f| %>
+  <%= f.hidden_field :restaurant_id, value: current_employee.restaurant_id %>
+
+  <div class="form-body">
+    <div class="form-group row">
+      <%= f.label :title,class:"control-label text-right col-md-3" %>

```

Comment:
Indentation!
`<%= f.label :title, class:"control-label text-right col-md-3" %>`

Reply:
Please! set indentation throughout the forms and other partials.

Reply:
done